job-view.php

```diff
@@ -1,91 +1,122 @@
                                         <?php
-                                        if (!empty($row['jobs_location'])) {
-                                            echo "Job location: " . $row['jobs_location'];
+                                        if ( ! empty( $job['jobs_location'] ) ) {
+	                                        echo "Job location: " . $job['jobs_location'];
                                         } else {
-                                            echo "No location specified";
+	                                        echo "No location specified";
                                         }
                                         ?>
                                     </span>
-                                    <span class="job-type">Contract staff:</span>
-                                    <span class="job-price"><?php echo $row['salary']; ?> лв.</span>
-                                </div>
-                            </header>
-                            <div class="job-body">
-                                <?php
-                                echo $row['description'];
-                                ?>
+                                <span class="job-type">Contract staff:</span>
+                                <span class="job-price"><?php echo $row['salary']; ?> лв.</span>
                             </div>
+                        </header>
+                        <div class="job-body">
+							<?php
+							echo $job['description'];
+							?>
                         </div>
                     </div>
                 </div>
-                <aside class="job-secondary">
-                    <div class="job-logo">
-                        <div class="job-logo-box">
-                            <?php
-                            $img_file_logo = "./img/company/" . $row['id_user'] . ".jpg";
-                            if (file_exists($img_file_logo)) {
-                                echo '<img src="' . BASE_URL . "/img/company/" . $row['id_user'] . ".jpg" . '" alt="">';
-                            } else {
-                                echo '<img src="' . BASE_URL . "/img/company/" . "404.gif" . '" alt="">';
-                            }
-                            ?>
-                        </div>
-                    </div>
-                    <a href="#" class="button button-wide">Apply now</a>
-                    <?php
-                    if (!empty($row['company_site'])) { ?>
-                        <a href="<?php echo "http://" . $row['company_site']; ?>"
-                           target="_blank"><?php echo $row['company_site']; ?></a>
-                    <?php }
-                    ?>
-                </aside>
             </div>
+            <aside class="job-secondary">
+                <div class="job-logo">
+                    <div class="job-logo-box">
+						<?php
+						$img_file_logo = "./img/company/" . $job['id_user'] . ".jpg";
```

Comment:
what would happen if the image is not `.jpg`, but a `.png` file?

Reply:
We can't have PNG or any other graphic format.
The idea is to keep some consistency.
When a client uploads another format other than JPG it will be converted on upload